install/0000_00_cluster-version-operator_03_usc_deployment-DevPreviewNoUpgrade.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: update-status-controller
+  namespace: openshift-update-status-controller
+  annotations:
+    kubernetes.io/description: The update status controller manages OpenShift UpdateStatus API from insights collected from cluster components.
+    include.release.openshift.io/self-managed-high-availability: "true"
+    release.openshift.io/feature-set: DevPreviewNoUpgrade
+spec:
+  selector:
+    matchLabels:
+      k8s-app: update-status-controller
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      name: update-status-controller
+      annotations:
+        target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
+        openshift.io/required-scc: restricted-v2
+      labels:
+        k8s-app: update-status-controller
+    spec:
+      securityContext:
+        runAsNonRoot: true
+      containers:
+      - name: update-status-controller
+        image: {{.ReleaseImage}}
+        imagePullPolicy: IfNotPresent
+        args:
+        - "update-status-controller"
+        - -v=5 # High while in DevPreview, lower for GA
+        securityContext:
+          capabilities:
+            drop:
+            - ALL
+          allowPrivilegeEscalation: false
+        resources:
+          requests:
+            cpu: 20m
+            memory: 50Mi
+        terminationMessagePolicy: FallbackToLogsOnError
+      nodeSelector:
```

Comment:
Hmm, the build02 does not have a node with such a role. This led me to https://access.redhat.com/solutions/7028754 and https://issues.redhat.com/browse/OCPBUGS-16964. So applying this on an older cluster with a new version would fail right now, I suppose.

Reply:
My attempt to add a multi-line comment using Goland has failed. `node-role.kubernetes.io/control-plane` is the role.

Reply:
Disappointing. I'll revert this to `master`